final_prep/README.md

```diff
@@ -33,23 +34,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="748" alt="Screen Shot 2021-04-11 at 9 44 00 PM" src="https://user-images.githubusercontent.com/79363697/114337887-30c58d80-9b0f-11eb-8686-d6199ad34500.png">
 - `Add Week 2 Screenshot Here`
+<img width="751" alt="Screen Shot 2021-04-11 at 9 44 09 PM" src="https://user-images.githubusercontent.com/79363697/114337981-5c487800-9b0f-11eb-843e-578827e11904.png">
```

Comment:
I would work on filling these out a bit. You can sync up with the Mod1 schedule, which will help. In my experience, a full day off of Turing work was very rare. Of course, everyone's situation is different, and I had a different experience, attending Turing on campus, but just want to prepare you for project work taking plenty of weekend time. That being said, I always tried to take more time for myself on weekends than on a weekday.